src/views/Dashboard/Customer/Home/MyTest/TestResult.vue

```diff
@@ -81,45 +88,232 @@
 </template>
 
 <script>
+import ipfsWorker from "@/common/lib/ipfs/ipfs-worker";
+import { mapState } from "vuex";
+import { queryDnaTestResults } from "@/common/lib/polkadot-provider/query/genetic-testing";
+import { queryLabsById } from "@/common/lib/polkadot-provider/query/labs";
+import { getOrdersDetail } from "@/common/lib/polkadot-provider/query/orders";
+import { queryServicesById } from "@/common/lib/polkadot-provider/query/services";
+import { hexToU8a } from "@polkadot/util";
+import { submitRatingOrder } from "@/common/lib/rating";
 import { downloadIcon, debioIcon, creditCardIcon, starIcon, checkCircleIcon } from "@/common/icons"
 import Modal from "@/common/components/Modal"
+import Rating from "@/common/components/Rating"
 export default {
   name: "TestResult",
-  components: { Modal },
+  components: { 
+    Modal,
+    Rating
+  },
   data: () => ({
+    privateKey: "",
+    publicKey: "",
+    specimentNumberInput: "",
+    ownerAddress: "",
+    speciment: {},
+    services: [],
+    rating: 0,
+    lab: null,
+    order: null,
+    isDataPdf: false,
+    serviceName: "",
+    serviceCategory: "",
```

Comment:
Please separate `name`, `components`, and `data` with white spaces in between while you're at it. Also, refer to our doc: https://blocksphere2020.atlassian.net/wiki/spaces/DBIO/pages/79724545/Frontend+Code+Convention#Add-in-new-lines-between-functions